test/integration/network-e2e/sample-transaction-event-handler.js

```diff
@@ -144,11 +144,12 @@ class SampleTransactionEventHandler {
  * @param {Transaction} transaction The transaction being submitted.
  * @param {Network} network The network where the transaction is being submitted.
```

Comment:
This parameter now needs to be documented as `DefaultEventHandlerOptions` type instead of `Network` to match the function implementation